gustaf/show.py

```diff
@@ -43,6 +50,24 @@ def __call__(self, *args, **kwargs):
 sys.modules[__name__].__class__ = _CallableShowDotPy
 
 
+def is_ipython():
+    """Returns True if the current environment is IPython.
+
+    Check if the code is run in a notebook.
+
+    Returns
+    -------
+    bool
+        True if the current environment is IPython.
+    """
+    try:
+        from IPython import get_ipython
+
+        return get_ipython() is not None
+    except ImportError:
+        return False
+
+
```

Comment:
can we just do this once and save this as a variable? or is there any situation where this changes dynamically?

Reply:
Done